staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_proxy_test.go

```diff
@@ -376,6 +378,8 @@ func TestProxyUpgrade(t *testing.T) {
 		},
 	}
 
+	utilfeature.DefaultFeatureGate.Add(features.DefaultKubernetesFeatureGates)
```

Comment:
why?

Reply:
Since that test is never calling `(ServerRunOptions) ApplyTo`, the DefaultFeatureGate is never initialized properly. If the feature-gate are not initialized, then the test panics later because it can't find some required feature.

Reply:
I think I can fix that, I'll update the PR and tell me what you think.